docs/Rakefile

```diff
@@ -78,7 +78,7 @@ task :adoc_tar => [:init, :adoc_variables, :markdown_to_asciidoc] do
       file_list.each { |f|
         tar.add_file(f, 0444) { |io|
           if File.extname(f) == ".adoc" then
-            adoc_content = File.read(f)
+            adoc_content = File.read(f, encoding: "ISO8859-1:utf-8")
```

Comment:
Ok, I see now the other error report you posted. Where is this coming from? I have not seen this before.